src/share/classes/com/sun/javatest/regtest/agent/JUnitRunner.java

```diff
@@ -25,17 +25,27 @@
 
 package com.sun.javatest.regtest.agent;
 
+import org.junit.platform.engine.discovery.DiscoverySelectors;
+import org.junit.platform.launcher.Launcher;
+import org.junit.platform.launcher.LauncherDiscoveryRequest;
+import org.junit.platform.launcher.LauncherSession;
+import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
+import org.junit.platform.launcher.core.LauncherFactory;
+import org.junit.platform.launcher.listeners.SummaryGeneratingListener;
+import org.junit.platform.launcher.listeners.TestExecutionSummary;
+
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.lang.reflect.Method;
 
-
 /**
  * TestRunner to run JUnit tests.
  */
 public class JUnitRunner implements MainActionHelper.TestRunner {
     private static final String
-        JUNIT_NO_DRIVER        = "No JUnit 4 driver (install junit.jar next to jtreg.jar)";
+        JUNIT_NO_DRIVER        = "No JUnit driver (install junit-platform-console-standalone.jar next to jtreg.jar)";
```

Comment:
The code should not specifically require this jar file by name. We went to an effort recently to decouple the names of the jar files used to build jtreg from the code that uses them.
Admittedly, this string literal was an accidental oversight from that recent work. I suggest changing the name of the jar file to just `JUnit` or `JUnit jar file(s)`

Reply:
Changed to `JUnit JAR file(s)`.